pyswarms/backend/topology/random.py

```diff
@@ -206,7 +203,7 @@ def __compute_neighbors(self, swarm, k):
         # Generate connected graph.
         while connected_components(adj_matrix, directed=False, return_labels=False) != 1:
             for i, j in itertools.product(range(swarm.n_particles), repeat=2):
-                    if dist_matrix[i][j] == 0:
+                    if dist_matrix[i][j] == np.inf:
```

Comment:
This is the bug in the `Random` class right? :+1:

Reply:
That's right 👍